mesh/src/test/java/no/nordicsemi/android/mesh/sensorutils/DevicePropertyCharacteristicTest.java

```diff
@@ -267,4 +267,43 @@ public void power() {
             }
         }
     }
+
+    @Test
+    public void electricCurrent() {
+        final ArrayList<Float> expectedSamples = new ArrayList<>();
+        expectedSamples.add(0.0f);
+        expectedSamples.add(0.01f);
+        expectedSamples.add(327.67f);
+        expectedSamples.add(655.34f);
+        expectedSamples.add(null);
+        final ArrayList<byte[]> samples = new ArrayList<>();
+        // Note that sample data has been padded with one "random" byte to test offset.
```

Comment:
The samples also have a padded byte at the end as well.

Reply:
Yes, will fix.